mlrun/config.py

```diff
@@ -264,6 +264,12 @@
                     "executing": "24h",
                 }
             },
+            "retry": {
+                "backoff": {
+                    "default_base_delay": "30s",
```

Comment:
is the `base` redundant?

Reply:
it's base delay because the actual delay is `f(base_delay, attempt)`